pkg/kubelet/stats/stats_provider_test.go

```diff
@@ -556,8 +556,6 @@ func generateCustomMetricSpec() []cadvisorapiv1.MetricSpec {
 				e.Type = cadvisorapiv1.MetricGauge
 			case 1:
 				e.Type = cadvisorapiv1.MetricCumulative
-			case 2:
-				e.Type = cadvisorapiv1.MetricDelta
```

Comment:
Hmmm... You removed this in https://github.com/google/cadvisor/pull/2445, but I think it may be possible to use it, since we  cast a string to a metric type here: https://github.com/google/cadvisor/blob/master/collector/prometheus_collector.go#L153

Reply:
Ah right. please check the new update to see if it is enough. thanks @dashpole